src/test/App.test.js

```diff
@@ -36,9 +41,68 @@ describe("App", () => {
     props = playerTurn.props();
     expect(props.player).toBe(PLAYER.X);
   });
-  it("can determine X as winner", () => {});
-  it("can determine a tie", () => {});
-  it("can determine O as winner", () => {});
-  it("can determine when the game is over", () => {});
-  it("can start a new game", () => {});
+
+  it("can determine X as winner", () => {
+    const wrapper = mount(<App />);
+    const square = wrapper.find(Square);
+    square.at(0).simulate("click"); //X
+    expect(wrapper.find(EndGameDeclaration)).toHaveLength(0);
+
+    square.at(3).simulate("click"); //O
+    square.at(1).simulate("click"); //X
+    square.at(5).simulate("click"); //O
+    square.at(2).simulate("click"); //X
+    // console.log(wrapper.debug());
+    const endGame = wrapper.find(EndGameDeclaration);
+    // console.log(endGame.debug());
+    expect(endGame.text()).toBe("Player X Won!");
+  });
+
+  it("can determine O as winner", () => {
+    const wrapper = mount(<App />);
+    const square = wrapper.find(Square);
+    square.at(3).simulate("click"); //X
+    expect(wrapper.find(EndGameDeclaration)).toHaveLength(0);
+    square.at(0).simulate("click"); //O
+    square.at(5).simulate("click"); //X
+    square.at(1).simulate("click"); //O
+    square.at(6).simulate("click"); //X
+    square.at(2).simulate("click"); //0
+    const endGame = wrapper.find(EndGameDeclaration);
+    expect(endGame.text()).toBe("Player O Won!");
+  });
+  it("can determine a tie", () => {
+    const wrapper = mount(<App />);
+    const square = wrapper.find(Square);
+    square.at(0).simulate("click"); //X
+    expect(wrapper.find(EndGameDeclaration)).toHaveLength(0);
+    square.at(2).simulate("click"); //O
+    square.at(1).simulate("click"); //X
+    square.at(3).simulate("click"); //O
+    square.at(5).simulate("click"); //X
+    square.at(4).simulate("click"); //0
+    square.at(6).simulate("click"); //X
+    square.at(8).simulate("click"); //0
+    square.at(7).simulate("click"); //X
+    const endGame = wrapper.find(EndGameDeclaration);
+    expect(endGame.text()).toBe("Cat's Game!");
+  });
+  it("can determine when the game is over", () => {
+    // is this test needed with the test above?
+  });
+  it("can start a new game", () => {
+    const wrapper = mount(<App />);
+    const square = wrapper.find(Square);
+    square.at(3).simulate("click"); //X
+    expect(wrapper.find(EndGameDeclaration)).toHaveLength(0);
+    square.at(0).simulate("click"); //O
+    square.at(5).simulate("click"); //X
+    square.at(1).simulate("click"); //O
+    square.at(6).simulate("click"); //X
+    square.at(2).simulate("click"); //0
+    const endGame = wrapper.find(EndGameDeclaration);
+    expect(endGame.text()).toBe("Player O Won!");
+    const newGame = wrapper.find(NewGame);
+    newGame.find("Button").simulate("click");
```

Comment:
You need to assert a new game has started by the Player being X and the board is blank